src/ape/types/trace.py

```diff
@@ -225,3 +230,428 @@ class TraceFrame(BaseInterfaceModel):
     """
     The raw trace frame from the provider.
     """
+
+
+class ControlFlow(BaseModel):
```

Comment:
Does this class have ape-specific functionality?

Reply:
i thought about moving to `ethpm-types` but decided against it because it really is only about execution. it felt weird putting anything that is solely about execution in ethpm-types for some reason, probably because there wasnt good spot for it on its own.

but the answer is no, it is basically a class that only works with classes from ethpm-types

Reply:
it is kindof like the main helper class to the source traceback too, they go kind of hand in hand and make the most sense together, at least that is what i think currently